commit/solvers.py

```diff
@@ -264,14 +264,14 @@ def solve(y, A, At, tol_fun = 1e-4, tol_x = 1e-6, max_iter = 1000, verbose = 1,
     if x0 is None:
         x0 = np.zeros(A.shape[1])
 
-    return fista( y, A, At, tol_fun, tol_x, max_iter, verbose, x0, omega, prox)
+   
+    return fista( y, A, At, tol_fun, tol_x, max_iter, verbose, x0, omega, prox, confidence_map)
 
-
-def fista( y, A, At, tol_fun, tol_x, max_iter, verbose, x0, omega, proximal) :
+def fista( y, A, At, tol_fun, tol_x, max_iter, verbose, x0, omega, proximal, confidence_map) :
     """
     Solve the regularised least squares problem
 
-        argmin_x 0.5*||Ax-y||_2^2 + Omega(x)
+        argmin_x 0.5*|| W ( Ax-y ) ||_2^2 + Omega(x)
```

Comment:
Are you sure about this equation? Shouldn't _W_ be _\sqrt(W)_? See here: https://en.wikipedia.org/wiki/Weighted_least_squares

Reply:
Yes, it should be sqrt(W)